tests/setup_test.go

```diff
@@ -6768,3 +6770,444 @@ func TestGetProjectRunWebhookDeliveries(t *testing.T) {
 		}
 	})
 }
+
+func TestProjectRunWebhookRedelivery(t *testing.T) {
+	t.Parallel()
+
+	config := `
+		{
+			runs: [
+			{
+				name: 'run01',
+				tasks: [
+				{
+					name: 'task01',
+					runtime: {
+					containers: [
+						{
+						image: 'alpine/git',
+						},
+					],
+					},
+					steps: [
+					{ type: 'run', command: 'env' },
+					],
+				},
+				],
+			},
+			],
+		}
+	`
+
+	t.Run("test redelivery project run webhook delivery with deliverystatus = deliveryError", func(t *testing.T) {
+		dir := t.TempDir()
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		sc := setup(ctx, t, dir, withGitea(true), withWebhooks("testWebhookURL", webhookSecret))
```

Comment:
is this done to make the delivery fail? So, this should be clearer since now it's not documented and the url doesn't explain this.